examples/features/observability/README.md

```diff
@@ -0,0 +1,3 @@
+This example is the Hello World example instrumented for logs, metrics, and tracing.
+
+Please refer to Microservices Observability user guide for setup.
```

Comment:
Add a link here?

Reply:
I guess that's true.. we have a user guide but it doesn't have the instructions for running this yet.